cms/settings/base.py

```diff
@@ -917,3 +919,42 @@
 # FIXME: remove before going live
 ENFORCE_EXCLUSIVE_TAXONOMY = env.get("ENFORCE_EXCLUSIVE_TAXONOMY", "true").lower() == "true"
 ALLOW_TEAM_MANAGEMENT = env.get("ALLOW_TEAM_MANAGEMENT", "false").lower() == "true"
+
+# Auth
+WAGTAIL_CORE_ADMIN_LOGIN_ENABLED = env.get("WAGTAIL_CORE_ADMIN_LOGIN_ENABLED", "false").lower() == "true"
+LOGOUT_REDIRECT_URL = env.get("LOGOUT_REDIRECT_URL", env.get("WAGTAILADMIN_LOGIN_URL"))
+AUTH_TOKEN_REFRESH_URL = env.get("AUTH_TOKEN_REFRESH_URL")
+SESSION_COOKIE_AGE = env.get("SESSION_COOKIE_AGE", 60 * 15)  # 15 minutes to match Auth Service
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
+IDENTITY_API_BASE_URL = env.get("IDENTITY_API_BASE_URL")
+AWS_COGNITO_LOGIN_ENABLED = env.get("AWS_COGNITO_LOGIN_ENABLED", "true").lower() == "true"
+AWS_COGNITO_USER_POOL_ID = env.get("AWS_COGNITO_USER_POOL_ID")
+AWS_COGNITO_APP_CLIENT_ID = env.get("AWS_COGNITO_APP_CLIENT_ID")
+
+# TODO: Make system checks
+# if AWS_COGNITO_LOGIN_ENABLED:
+#     required_env_vars = [
+#         "IDENTITY_API_BASE_URL",
+#         "AUTH_TOKEN_REFRESH_URL",
+#         "AWS_COGNITO_APP_CLIENT_ID",
+#         "AWS_COGNITO_USER_POOL_ID",
+#     ]
+#     for var in required_env_vars:
+#         if not env.get(var):
+#             raise ImproperlyConfigured(f"Missing required environment variable: {var}")
+
+
+# Groups
+PUBLISHING_ADMIN_GROUP_NAME = "Publishing Admins"
+PUBLISHING_OFFICER_GROUP_NAME = "Publishing Officers"
+VIEWERS_GROUP_NAME = "Viewers"
+ROLE_GROUP_IDS = {"role-admin", "role-publisher"}
+
+# Cookie Names
+ACCESS_TOKEN_COOKIE_NAME = "access_token"  # noqa: S105
+REFRESH_TOKEN_COOKIE_NAME = "refresh_token"  # noqa: S105
+ID_TOKEN_COOKIE_NAME = "id_token"  # noqa: S105
```

Comment:
Maybe we can add a comment here saying that this name is used explicitly in the JS, and so if it gets changed, the JS needs to be updated? Unless we think this is so unlikely that we don't need to bother.

Reply:
I've made it use the settings value, so it is not out of sync. Though in the JS lib, it does hardcode it, so we would need to change it in the app if the JS lib changes, but at least it is in one place for us to change now. [`19cd39d` (#163)](https://github.com/ONSdigital/dis-wagtail/pull/163/commits/19cd39de350fcc17519fe05f42b38c54af137d7a)